src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -1929,17 +1949,23 @@ public virtual Task<IList<TUser>> GetUsersInRoleAsync(string roleName)
         /// <param name="user"></param>
         /// <param name="methodName"></param>
         /// <returns>result</returns>
-        protected async Task<bool> LogResultAsync(bool result,
+        protected virtual async Task<bool> LogResultAsync(bool result,
             TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            var baseMessage = Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user));
-            if (result)
-            {
-                Logger.WriteInformation(string.Format("{0} : {1}", baseMessage, result.ToString()));
-            }
-            else
+            var logLevel = result ? LogLevel.Information : LogLevel.Warning;
+
+            if (Logger.IsEnabled(logLevel))
             {
-                Logger.WriteWarning(string.Format("{0} : {1}", baseMessage, result.ToString()));
+                var baseMessage = Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user));
+
+                if(logLevel == LogLevel.Warning)
```

Comment:
sure Logger.Write(LogLevel.Information, eventId, message, null, null) should work the same but yes not easily super concise. On the flip side, if we make the eventId configurable, then users can set specific eventId for Identity messages something not possible right now. Console logger doesn't much care about the eventId but other loggers might